### ISSUE TITLE
关于在特权观测里面添加高度信息（self.measured_heights）报错

### ISSUE THREAD

**Issue by wuyukun-tong** [limxdynamics/pointfoot-legged-gym#1]
当我尝试将self.measured_heights添加到特权观测中的时候
 def compute_group_observations(self):
        # note that observation noise need to modified accordingly !!!
        dof_list = [0,1,2,4,5,6]
        dof_pos = (self.dof_pos - self.default_dof_pos)[:,dof_list]
        # dof_pos = torch.remainder(dof_pos + self.pi, 2 * self.pi) - self.pi

        obs_buf = torch.cat( 
        )
        critic_obs_buf = torch.cat((
            self.base_lin_vel * self.obs_scales.lin_vel, self.obs_buf), 
            self.measured_heights, 
            dim=-1)
        return obs_buf, critic_obs_buf
self.measured_heights打印出来为0，是不是调用的顺序有问题
**Expected behavior**

**System (please complete the following information):**
 - Commit: [e.g. 8f3b9ca]
 - OS: [e.g. Ubuntu 22.04]
 - GPU: [e.g. RTX  4090]
 - CUDA: [e.g. 12.4]
 - GPU Driver: [e.g. 550]

**Additional context**
